lib/block_view_stake.go

```diff
@@ -1080,7 +1106,7 @@ func (bav *UtxoView) _connectStake(
 	if err != nil {
 		return 0, 0, nil, errors.Wrapf(err, "_connectStake: ")
 	}
-	if prevValidatorEntry == nil || prevValidatorEntry.isDeleted || prevValidatorEntry.DisableDelegatedStake {
+	if prevValidatorEntry == nil || prevValidatorEntry.isDeleted {
```

Comment:
This was a duplicate check. We already check for this in `IsValidStakeMetadata`.